src/AddTodoForm.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 
-function AddTodoForm() {
+function AddTodoForm(props) {
   const handleAddTodo = (event) => {
-    event.preventDefault();
     const todoTitle = event.target.title.value;
+    props.onAddTodo(todoTitle);
     console.log(todoTitle);
+    event.preventDefault();
```

Comment:
I would just move this line `event.preventDefault` to line five to avoid any unintended behavior. This is my personal opinion because by putting int at the top of the function I don't have to think about the order of operations.

Reply:
Thank you so much for the feedback! 
I will push event.preventDefault to line 5 before merging it to the main branch.